dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
```

Comment:
What about using local storage on the client side to track whether user has closed dialog or not (we do this for some things already I think, and I plan to also use that approach for alerts that warn that we're not saving progress during hour of code)